### PR TITLE
add github actions

### PR DESCRIPTION
**Pull request by pazembrz**

### CODE REVIEW

**Comment by michamos** [comment]
secret is missing, so it can't be deployed to PyPI. @drjova I saw you created repo-level secrets. Wouldn't it be easier to handle those at the org level?

**Comment by drjova** [comment]
unless I'm missing something on pypi you can create a token per repo

**Comment by michamos** [comment]
https://pypi.org/help/#apitoken
> You can create a token for an entire PyPI account, in which case, the token will work for all projects associated with that account. Alternatively, you can limit a token's scope to a specific project.

**Comment by drjova** [comment]
Screenshot 2020-12-02 at 15 43 14
👍  I didn't notice the first option

File: .github/workflows/build-and-release.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        run: |
+          python setup.py sdist bdist_wheel
+
+      - name: Publish package
+        if: ${{ success() && github.event_name == 'push' && matrix.python-version == '2.7'  }}
+        uses: pypa/gh-action-pypi-publish@v1.3.1
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
+
```
[the remark this answers is not in the document]
Comment (pazembrz) [reply .github/workflows/build-and-release.yml:73]: If we want Coveralls we have to do this like that as we have matrix for python versions.

**Review by drjova — APPROVED** [review]

**Pull request merged**